PackageInfo.g

```diff
@@ -1,9 +1,10 @@
 SetPackageInfo( rec(
 
+Packagename := "kbmag",
 PackageName := "kbmag",
```

Comment:
You inserted a second `PackageName`

Reply:
From who knows how long ago I remember an instruction to define "Packagename" as well as "PackageName" in a PackageInfo.g (which is what happens in the various Bangor packages). No doubt it is no longer necessary (if it ever was) so the lowercase version can go.